versions.props

```diff
@@ -1,4 +1,5 @@
 com.fasterxml.jackson.*:* = 2.8.11
```

Comment:
btw is the `b-duplicate-different-versions-1.0.jar` necessary?

Reply:
That is used, I just changed one of the versions in there to make them not compatible, such that the merger would kick in